WeakAuras/Bindings.xml

```diff
@@ -2,13 +2,7 @@
   <Binding name="WEAKAURASTOGGLE" header="WEAKAURAS" Category="ADDONS">
     WeakAuras.OpenOptions()
   </Binding>
-  <Binding name="WEAKAURASSTARTPROFILING" Category="ADDONS">
-    WeakAuras.StartProfile()
-  </Binding>
-  <Binding name="WEAKAURASSTOPPROFILING" Category="ADDONS">
-    WeakAuras.StopProfile()
-  </Binding>
-  <Binding name="WEAKAURASPRINTPROFILING" Category="ADDONS">
-    WeakAuras.PrintProfile()
+  <Binding name="WEAKAURASPROFILINGTOGGLE" Category="ADDONS">
+    WeakAuras.RealTimeProfilingWindow:Toggle()
```

Comment:
I think we should keep the print binding.

Reply:
yea

Reply:
done